package/kedro_viz/database.py

```diff
@@ -21,21 +21,3 @@ def configure_wal_for_azure(engine):
     if is_azure_ml:
         with engine.connect() as conn:
             conn.execute(text("PRAGMA journal_mode=WAL;"))
-
```

Comment:
This file is for handling SQLite which is not needed anymore, so I guess we can delete the file entirely